app/dcs/api/uplink-cli/access-command/access-register/page.md

```diff
@@ -27,11 +27,11 @@ uplink access register <flags> <ACCESS-GRANT>
 
 ## Flags
 
-| Flag                    | Description                                                      |
-| :---------------------- | :--------------------------------------------------------------- |
-| `--access string`       | the serialized access, or name of the access to use              |
-| `--auth-service string` | the address to the service you wish to register your access with |
-| `--public`              | if the access should be public. Default false                    |
+| Flag                    | Description                                                                                          |
+| :---------------------- | :--------------------------------------------------------------------------------------------------- |
+| `--access string`       | the serialized access, or name of the access to use                                                  |
+| `--auth-service string` | the address to the service you wish to register your access with (default "auth.storjshare.io:7777") |
```

Comment:
Is the port actually defined in the default?

Reply:
yeah, that's the DRPC endpoint this particular command uses. I'm not sure why it's not using https, but if you run `uplink access register -h` you'll see the default used